src/main/java/uk/gov/hmcts/darts/retention/controller/RetentionController.java

```diff
@@ -43,9 +43,10 @@ public ResponseEntity<List<GetCaseRetentionsResponse>> retentionsGet(Integer cas
             securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA, ADMIN},
             globalAccessSecurityRoles = {JUDGE, ADMIN})
     @Override
-    public ResponseEntity<PostRetentionResponse> retentionsPost(PostRetentionRequest postRetentionRequest) {
+    public ResponseEntity<PostRetentionResponse> retentionsPost(Boolean validateOnly,
```

Comment:
minor: shall we add `@RequestParam` ? just to be consistent with what we do in other controllers

Reply:
there was actually a comment one time to not do that, as it appears in the RetentionApi that we are implementing (auto generated in there) and so we were asked not to duplicate it in the implementation.